hanson_at_fb2_iterative.py

```diff
@@ -0,0 +1,30 @@
+# given a list of unqiue, integers, return all possible subsets
+# input: [1, 2, 3]
+# output: [[1], [2], [3], [1, 2], [1, 3], [2, 3], [1,2,3] []]
+
+# base case, input [], output []
+# input [1], output [], [1]
+# input [2], output [], [2], [1], [1, 2]
+# every time you add the new number to previously existing elements
+
+def subset_list (input_list):
+	output_list = [[]]
```

Comment:
I think all these variable names could be named more concise (don't need _list. maybe use pluralizations instead) and more clear (e.g. how does total, storage, temp_list_within differ form each other?) No good naming suggestions for you right now because I'm taking a quick look, but maybe later